.github/scripts/badwords.pl

```diff
@@ -56,13 +56,13 @@ sub file {
         foreach my $w (@w) {
             my $case = $exactcase{$w};
             if(($in =~ /^(.*)$w/i && !$case) ||
-               ($in =~ /^(.*)$w/ && $case) ) {
+                ($in =~ /^(.*)$w/ && $case) ) {
                 my $p = $1;
                 my $c = length($p)+1;
                 print STDERR  "$f:$l:$c: error: found bad word \"$w\"\n";
                 printf STDERR " %4d | $in\n", $l;
                 printf STDERR "      | %*s^%s\n", length($p), " ",
-                    "~" x (length($w)-1);
+                  "~" x (length($w)-1);
```

Comment:
We use a 4-space indent but here the script wants 2-space indents for continued lines (this is seen on multiple places). It does not seem to match how we write code. (and goes against what emacs perl-mode does by default)